### ISSUE TITLE
Release action failed: 'packaged_addon/*.nvda-addon': No such file or directory

### ISSUE THREAD

**Issue by hwf1324** [nvdaaddons/AddonTemplate#65]
I don't know why the uploaded Artifacts are not in the directory?

see: https://github.com/hwf1324/mouseEnhancement/actions/runs/17060085888/job/48365202624#step:5:1
